### PR TITLE
DOC: update the window.Rolling.min docstring

### PR DESCRIPTION
**Pull request by gojandrooo**
Checklist for the pandas documentation sprint (ignore this if you are doing
an unrelated PR):

- [ ] PR title is "DOC: update the <your-function-or-method> docstring"
- [ ] The validation script passes: `scripts/validate_docstrings.py <your-function-or-method>`
- [ ] The PEP8 style check passes: `git diff upstream/master -u -- "*.py" | flake8 --diff`
- [ ] The html version looks good: `python doc/make.py --single <your-function-or-method>`
- [ ] It has been proofread on language by another sprint participant

Please include the output of the validation script below between the "```" ticks:

```
# paste output of "scripts/validate_docstrings.py <your-function-or-method>" here
# between the "```" (remove this comment, but keep the "```")

```

If the validation script still gives errors, but you think there is a good reason
to deviate in this case (and there are certainly such cases), please state this
explicitly.

Checklist for other PRs (remove this part if you are doing a PR for the pandas documentation sprint):

- [ ] closes #xxxx
- [ ] tests added / passed
- [ ] passes `git diff upstream/master -u -- "*.py" | flake8 --diff`
- [ ] whatsnew entry

### CODE REVIEW

File: pandas/core/window.py [hunk 1]
```diff
@@ -838,9 +838,48 @@ def max(self, *args, **kwargs):
...
+    Series.min : Similar method for Series
+    DataFrame.min : Similar method for DataFrame
+
+    Examples
+    --------
+    The below example will show a rolling calculation
+    with a window size of 3.
+
+    >>> s = pd.Series([4,3,5,2,6])
+    >>> s
```
Comment (WillAyd) [inline pandas/core/window.py]: The constructor here is simple enough that you don't need to explicitly print

**Review by WillAyd — CHANGES_REQUESTED** [review]

File: pandas/core/window.py [hunk 2]
```diff
@@ -838,9 +838,48 @@ def max(self, *args, **kwargs):
...
+    DataFrame.%(name)s : Calling object with a DataFrame
+    Series.min : Similar method for Series
+    DataFrame.min : Similar method for DataFrame
+
+    Examples
+    --------
+    The below example will show a rolling calculation
+    with a window size of 3.
+
+    >>> s = pd.Series([4,3,5,2,6])
```
Comment (jorisvandenbossche) [inline pandas/core/window.py]: spaces after comma for PEP8

**Review by jorisvandenbossche — COMMENTED** [review]
Small comment + @WillAyd 's one, looks good!

**Review by jorisvandenbossche — APPROVED** [review]
Pushed a small update fixing the PEP8 issues

**Comment by jorisvandenbossche** [comment]
@gojandrooo Thanks!

**Pull request merged**